doc/whats_new/v1.2.rst

```diff
@@ -9,6 +9,17 @@ Version 1.2.1
 
 **In Development**
 
+
+Changes impacting all modules
+-----------------------------
+
+- |Fix| Fix a bug where the current configuration was ignored in estimators using
+  `n_jobs > 1`. This bug was triggered for tasks dispatched by the ancillary
+  thread of `joblib` as :func:`sklearn.get_config` used to access an empty thread
+  local configuration instead of the configuration visible from the thread where
+  `delayed` was first called.
```

Comment:
```suggestion
  `joblib.Parallel` was first called.
```

since `delayed` can be also be called in the ancillary thread when using generator expressions.